examples/platform/qpg/app/BaseAppTask.h

```diff
@@ -0,0 +1,134 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef BASE_APP_TASK_H
+#define BASE_APP_TASK_H
+
+/**********************************************************
+ * Includes
```

Comment:
do we need these comments?

Reply:
It is specific Qorvo coding style. Should we change it ?